test/intl402/NumberFormat/test-option-roundingPriority-mixed-options.js

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+  /*---
+  esid: sec-intl.numberformat.prototype.format
+  description: Tests that the digits are determined correctly when specifying at same time «"minimumFractionDigits", "maximumFractionDigits", "minimumSignificantDigits", "maximumSignificantDigits"»
+  features: [Intl.NumberFormat-v3]
+  includes: [testIntl.js]
+  ---*/
+
+  var locales = [new Intl.NumberFormat().resolvedOptions().locale, "ar", "de", "th", "ja"];
+
+  var numberingSystems =  ['arab', 'latn', 'thai', 'hanidec'];
+
+  var expectedResults = {
```

Comment:
Request: Please check the output of `resolvedOptions` as well.

Reply:
I will address this later on this same test or in a different set of tests